crates/editor_ui/Cargo.toml

```diff
@@ -20,18 +20,23 @@ space_prefab.workspace = true
 space_undo.workspace = true
 space_persistence.workspace = true
 space_shared.workspace = true
+serde.workspace = true
 
 bevy_egui = "0.24"
 egui-gizmo = "0.13"
 egui_dock = "0.9"
 
 bevy_debug_grid = "0.4"
 
-bevy_mod_picking = { version = "0.17", default-features = false, features = [
+bevy_mod_picking = { git = "https://github.com/dgriffith0/bevy_mod_picking.git", branch = "pick_hidden", default-features = false, features = [
     "backend_raycast",
     "selection",
 ] }
+bevy_asset_loader = "0.19"
 bevy_panorbit_camera = "0.9"
-bevy_sprite3d = "2.7.0"
+bevy_mod_billboard = "0.5.1"
```

Comment:
Nice library! Didn't you manage the same with the sprite3d?

Reply:
So thats the thing, sprite3d made the sprites require an extra step to rotate the sprites on each frame which then had some issues when the camera would go near the same y coord as the sprite.

This issue is completely fixed with the billboarding, where they always respect the camera location and rotation, but it also introduced the need for adding in the clickable (invisible) sphere that the bevy_mod_picking branch change is for.